protos/Robot_Echo/SRRobot.proto

```diff
@@ -484,7 +484,7 @@ PROTO SRRobot [
                   rotation 0 0 1 1.5708
                   children [
                     Cylinder {
-                      height 0.01
+                      height 0.05
                       radius 0.1
```

Comment:
Nit: It'd be great to have these be as close to the size of the wheel as possible, so it looks like they rotate and move correctly.

```suggestion
                      height 0.045
                      radius 0.095
```

![image](https://user-images.githubusercontent.com/6527489/105578939-89d91d00-5d7b-11eb-893c-f0be3d030690.png)

Reply:
The oversize diameter is so that the wheel doesn't visually sink through the floor once it has settled.